packages/components/header/src/less/header-tabs.less

```diff
@@ -106,13 +106,22 @@
   }
 
   &__container {
-    display: flex;
-    flex-wrap: nowrap;
     list-style: none;
     margin: @oui-header-tabs-margin;
     overflow: hidden;
     padding: 0;
     border-bottom: @oui-header-tabs-border-bottom;
+
+    ul& {
+      display: flex;
+      flex-wrap: nowrap;
+    }
+  }
+
+  &__list {
+    display: flex;
+    flex-wrap: nowrap;
+    transition: margin-left 0.25s ease;
```

Comment:
Is it possible to use `translateX()` instead of margins?

In that way we will prevent a to do a full layout/repaint/composite on the page.

Reply:
Yes, good idea. Checking for it